ci/docker/image-builder/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM openjdk:11 as BUILD
```

Comment:
we should provide an ability to use "inhouse" image here

Reply:
Agree.

I see only one option - copy this base image to the inhouse registry and build upon it.

I'd like to propose a simplification. Let's use _only_ inhouse registry in the whole build chain:

```
openjdk (dockerhub) --> inhouse openjdk --> image-builder

ubuntu (dockerhub) --> inhouse ubuntu --> android-builder\...
```

Rationale:

- Switching between registries while building seems more complex. We need to ensure every time that they are identical and updated simultaneously.
- Uploading to DockerHub complicates the whole process and is not automated yet.
- Last, but not least: I don't see for whom we support DockerHub. Images are by design reproducible and everyone could build them by themselves.

Reply:
I'd like to discuss it in MBS-12767 because I don't want to make half-measures only for image-builder.
We have the same issue in other images (emulator, android-builder, ...).